packages/smart-contracts/src/contracts/lib/RinkebyChainlinkAggregatorCaller.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.5.0;
+
+interface AggregatorInterface {
+  function latestAnswer() external view returns (int256);
+
+  function latestTimestamp() external view returns (uint256);
+
+  function latestRound() external view returns (uint256);
+
+  function getAnswer(uint256 roundId) external view returns (int256);
+
+  function getTimestamp(uint256 roundId) external view returns (uint256);
+
+  event AnswerUpdated(int256 indexed current, uint256 indexed roundId, uint256 updatedAt);
+  event NewRound(uint256 indexed roundId, address indexed startedBy, uint256 startedAt);
+}
+
+
+/**
+ * @title ChainlinkAggregatorCaller for Rinkeby
+ */
+contract ChainlinkAggregatorCaller {
+  enum FiatEnum {USD, AUD, CHF, EUR, GBP, JPY}
+  enum CryptoEnum {ETH, DAI, USDT, USDC, SUSD}
+
+  /**
+   * @notice Get chainlink aggregator for a cyrpto to ETH rate
+   * @param _currencyCrypto crypto currency wanted
+   */
+  function getChainlinkAggregatorCryptoToETH(CryptoEnum _currencyCrypto)
+    internal
+    pure
+    returns (AggregatorInterface)
+  {
+    // SUSD/ETH not supported on rinkeby
+    // USDC/ETH not supported on rinkeby
+    // USDT/ETH not supported on rinkeby
+    revert("crypto not supported");
+  }
+
+  /**
+   * @notice Get chainlink aggregator for a cyrpto to USD rate
+   * @param _currencyCrypto crypto currency wanted
+   */
+  function getChainlinkAggregatorCryptoToUsd(CryptoEnum _currencyCrypto)
+    internal
+    pure
+    returns (AggregatorInterface)
+  {
+    if (_currencyCrypto == CryptoEnum.ETH) {
+      // ETH/USD
+      return AggregatorInterface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
+    }
+    if (_currencyCrypto == CryptoEnum.DAI) {
+      // DAI/USD
+      return AggregatorInterface(0x2bA49Aaa16E6afD2a993473cfB70Fa8559B523cF);
+    }
+    revert("crypto not supported");
+  }
+
+  /**
+   * @notice Get chainlink aggregator for a fiat to USD rate
+   * @param _currencyFiat fiat currency wanted
+   */
+  function getChainlinkAggregatorFiatToUsd(FiatEnum _currencyFiat)
+    internal
+    pure
+    returns (AggregatorInterface)
+  {
+    if (_currencyFiat == FiatEnum.AUD) {
+      // AUD/USD
+      return AggregatorInterface(0x21c095d2aDa464A294956eA058077F14F66535af);
+    }
```

Comment:
would it be possible to make this contract administrable by its owner? otherwise  adding currencies and changing addresses will be a nightmare

Reply:
I would rather upload a new contract. 
Adding administration there would make the contract less secure and more expensive to call. 
We can open an IRL discussion about it :)